Show results count in course search

mbenadda opened the pull request:
## Purpose

We want to show the number of results a given combination of a text query (or none) and search filters returns.

## Proposal

We can simply show it at the top of the search results, and repeat it at the bottom if the list is long enough to warrant it.

Closes #455

sampaccoud commented:
I noticed 2 problems:

- On my laptop the bottom count would be useful as soon as the number of results is more than 8,
- The bottom count touches the last result glimpses:

![image](https://user-images.githubusercontent.com/1427165/56662316-fc4b3380-66a3-11e9-90aa-2cef31c3bf3f.png)

mbenadda commented:
> * On my laptop the bottom count would be useful as soon as the number of results is more than 8,
Changed `20` to `8`

> * The bottom count touches the last result glimpses:
Fixed
